### PR TITLE
Create glossary.md

### PR DESCRIPTION
**Pull request by jhdalek55**
This is the same list of terminology from the Standard. It is being placed in the Deployment Best Practices document at the suggestion of @iramcdonald. Please suggest additional terms that may be specific to Uptane deployment.

### CODE REVIEW

**Comment by jhdalek55** [comment]
I accidentally added the glossary directly to the Master. I can't seem to remove that commit, so this may be a redundant Pull Request

File: glossary.md [hunk 1]
```diff
@@ -5,3 +5,66 @@ css_id: glossary
 
 # Glossary
 
+### Conformance terminology
+The keywords MUST, MUST NOT, REQUIRED, SHALL, SHALL NOT, SHOULD, SHOULD NOT, RECOMMENDED, MAY, and OPTIONAL in this document are to be interpreted as described in {{RFC2119}}.
+
+In order to be considered Uptane-compliant, an implementation MUST follow all of these rules as specified in the document.
```
Comment (pattivacek) [inline glossary.md:11]: This line doesn't really make sense here.

File: glossary.md [hunk 2]
```diff
@@ -5,3 +5,66 @@ css_id: glossary
...
+
+*Repository:* A server containing metadata about images, and sometimes the images themselves. Other data may be stored on the repository to be accessed by ECUs during the update process.
+
+*Suppliers:* Independent companies to which vehicle manufacturers may outsource the production of ECUs. Tier-1 suppliers directly serve the manufacturers. Tier-2 suppliers are those that perform outsourced work for Tier-1 suppliers.
+
+*Vehicle Version Manifest:* A compilation of all ECU version reports on a vehicle. It serves as a master list of all images currently running on all ECUs in the vehicle.
+
+### Uptane role terminology
+
+These terms are defined in greater detail in {{roles}}.
```
Comment (pattivacek) [inline glossary.md:41]: This should be a link to the Standard, I guess?

File: glossary.md [hunk 3]
```diff
@@ -5,3 +5,66 @@ css_id: glossary
 
 # Glossary
 
+### Conformance terminology
+The keywords MUST, MUST NOT, REQUIRED, SHALL, SHALL NOT, SHOULD, SHOULD NOT, RECOMMENDED, MAY, and OPTIONAL in this document are to be interpreted as described in {{RFC2119}}.
```
Comment (pattivacek) [inline glossary.md:9]: The link here need to be updated, or this line should be removed as well. Are these keywords actually relevant here?

File: glossary.md [hunk 4]
```diff
@@ -5,3 +5,66 @@ css_id: glossary
...
+
+* *Root role:* Signs metadata that distributes and revokes public keys used to verify the Root, Timestamp, Snapshot, and Targets role metadata.
+
+* *Snapshot role:* Signs metadata that indicates which images the repository has released at the same time.
+
+* *Targets role:* Signs metadata used to verify the image, such as cryptographic hashes and file size.
+
+* *Timestamp role:* Signs metadata that indicates if there are any new metadata or images on the repository.
+
+### Acronyms and abbreviations
```
Comment (pattivacek) [inline glossary.md:55]: Most of these acronyms and abbreviations only appear in the Standard but not in the Deployment pages or elsewhere on the website. Does that matter?

**Review by pattivacek — COMMENTED** [review]
No big worries about the accidental push to master. It is actually possible to remove the commit, but I don't have the rights. I can walk you through it if you want, but honestly, it's probably okay to just move on.

I have a few comments, but overall, I think I'm still missing why we want to duplicate these terms here. Is the intention to remove them from the Standard? I don't want to have to maintain this list in two places.

File: glossary.md (hunk 1 above)
Comment (jhdalek55) [reply glossary.md:11]: Good catch. I'll remove it.
```suggestion

```

Comment (jhdalek55) [reply glossary.md:11]: I'm going to re-open a pull request here to address your comments.

File: glossary.md (hunk 3 above)
Comment (jhdalek55) [reply glossary.md:9]: I'm going to remove them for now. They probably don't have the same significance here.

File: glossary.md (hunk 2 above)
Comment (jhdalek55) [reply glossary.md:41]: I'm not sure the comment is needed.

File: glossary.md (hunk 4 above)
Comment (jhdalek55) [reply glossary.md:55]: Probably not. I'm going to remove them as well.

**Pull request closed without merging**